src/main/java/dev/pgm/events/utils/Components.java

```diff
@@ -0,0 +1,29 @@
+package dev.pgm.events.utils;
+
+import tc.oc.pgm.lib.net.kyori.adventure.text.Component;
+import tc.oc.pgm.lib.net.kyori.adventure.text.event.ClickEvent;
+import tc.oc.pgm.lib.net.kyori.adventure.text.format.NamedTextColor;
+import tc.oc.pgm.lib.net.kyori.adventure.text.format.Style;
+
+public class Components {
+
+  public static Component command(Style style, String command, String... args) {
```

Comment:
Also, if adding a utility method like this is would be good to add a little java doc explaining it